src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -180,9 +179,8 @@ public Importer(final Config config, final FcrepoClient.FcrepoClientBuilder clie
     private void loadBagProfile(final String bagProfile) {
         try {
             // load the bag profile
-            final URL url = this.getClass().getResource("/profiles/" + bagProfile + ".json");
-            final InputStream in = url == null ? new FileInputStream(bagProfile) : url.openStream();
-            final BagProfile profile = new BagProfile(in);
+            // todo: same as export for profile init
```

Comment:
Is there another ticket for completing this `todo`?

Reply:
Looking at this todo it looks like this still needs to be done for this PR. I updated the `BagProfile` constructor to use the built in profiles, so this should be migrated to use that method.